balena/models/device_os.py

```diff
@@ -68,14 +68,21 @@ def get_config(self, app_id, options):
             endpoint=self.settings.get('api_endpoint')
         )
 
-    def download(self, raw=None, **data):
+    def download(self, raw=False, is_dev_image=False, **data):
```

Comment:
Doesn't that mean that if someone so far was `2.41.1+rev1.dev`, then they will now get a prod image? Maybe we can infer the dev variant with a `/\bdev\b/` regex.

Reply:
Nope, they will get a dev image, actually I didn't expect they will pass version name with `.prod` or `.dev` (since I don't see that from the version on dashboard) so I add `is_dev_image` so they can get dev image if needed but looks like we don't need it here as we assume user will pass os_variant in version.

Removed that option and I will leave version as it is. (The SDK already has the regex to handle dev variant here https://github.com/balena-io/balena-sdk-python/blob/master/balena/models/device_os.py#L155-L169)